lib/purl_parser.rb

```diff
@@ -1,12 +1,13 @@
 # class used to represent a purl, used to parse information
```

Comment:
It looks like rails always requires `pathname` but I don't think it does that for other parts of stdlib. I can't find any documentation on this point though. Here's the pathname require: https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails.rb#L3